src/NAR/RunTime

```diff
@@ -0,0 +1,77 @@
+package NAR;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class RunTime {
```

Comment:
can you please rename the class to "RuntimeSystem" to stay consistant with the documentation